README.md

```diff
@@ -15,11 +15,17 @@
   limitations under the License.
 -->
 
-# Hudi
+# Apache Hudi (Incubating)
 Apache Hudi (Incubating) (pronounced Hoodie) stands for `Hadoop Upserts Deletes and Incrementals`. 
 Hudi manages the storage of large analytical datasets on DFS (Cloud stores, HDFS or any Hadoop FileSystem compatible storage).
 
-### Features
+<http://hudi.apache.org/>
+
+[![Build Status](https://travis-ci.org/apache/incubator-hudi.svg?branch=master)](https://travis-ci.org/apache/incubator-hudi)
+[![GitHub release](https://img.shields.io/github/release/apache/incubator-hudi.svg)](https://github.com/apache/incubator-hudi/releases)
```

Comment:
this is pointing to non-asf releases.. can we point to the apache release? or remove this?

Reply:
I refer to this project [incubator-iotdb](https://github.com/apache/incubator-iotdb), if you worry about it, I will remove it.

Reply:
update